src/ec/packages/ec-component-ecl-compliance/components/ecl-compliance-link.html.twig

```diff
@@ -0,0 +1,23 @@
+{#
+  Parameters:
+  - "link" (object)
+  - "icon" (object)
+#}
+
+{% extends "@ecl-twig/ec-component-ecl-compliance/ecl-compliance.html.twig" %}
+
+{% set not_compliant =
+  (link.label is not defined or link.label is empty) and (
+    (link.icon_position is not defined or link.icon_position is empty) or
+    (icon[0].name is not defined or icon[0].name is empty) or
```

Comment:
because this condition is specific, for the link component, if you look at the test, indeed, you see how potentially we could fire both the validations and get errors from both the components ;)

Reply:
Thanks for explanation, it make sense now.